packages/typespec-azure-vscode/src/extension.ts

```diff
@@ -0,0 +1,19 @@
+// The module 'vscode' contains the VS Code extensibility API
+// Import the module and reference it with the alias vscode in your code below
+import * as vscode from "vscode";
+import { TypespecExtensionId } from "./constant";
+
+export async function activate(context: vscode.ExtensionContext) {
+  // load and activate typespec extension
+  const typespecExtension = vscode.extensions.getExtension(TypespecExtensionId);
```

Comment:
Do we need to check explicitly? I think the dependency configure should be enough, isn't it?

Reply:
We have set the dependency in package.json, so when install typespec-azure-extension, we will install typespec-extension if it is not installed. But after that we will loose the control of typespec-extension. In case, the typespec-extension is uninstalled someday. Since typespec-azure-extension is fully work on typespec-extension, we need to verify the installation of the typespec-azure-extension and activate it .